app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -154,6 +154,16 @@ def x509_session?
     identity.piv_cac_enabled?
   end
 
+  def x509_presented
+    if IdentityConfig.store.x509_presented_hash_attribute_requested_issuers.include?(
```

Comment:
The way this configuration is currently set up, it requires that we add all of the existing affected service providers for them to have the legacy behavior. Is that what we're expecting?

Reply:
I suppose that makes sense if we're wanting to prevent _new_ integrations from receiving the broken value, just want to make sure we're planning to get that configuration prepped ahead of time.

Reply:
@aduth yes, that's correct. i have the list of 30 prepped. i wasn't sure if i should wait until i merge to add it or if i should add it before merge?

Reply:
I think it'd be fine to add whenever it's ready to avoid having to worry about timing it with the merge + deploy.